### PR TITLE
Jose paquian

### PR DESCRIPTION
**Pull request by alostnight**

### CODE REVIEW

**Review by AliciaPetgrave — APPROVED** [review]

File: components/Cards/index.js [hunk 1]
```diff
@@ -17,3 +17,43 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+axios
+  .get("https://lambda-times-backend.herokuapp.com/articles")
+  .then(response => {
+    Object.values(response.data.articles).forEach(item => {
+      item.forEach(items => {
```
Comment (wSedlacek) [inline components/Cards/index.js:24]: The variable names here are a bit strange.
Doing a `item.forEach(items => {});` typically your plural is on the left of the forEach.
Beyond that I think better names could be used then `item` or `items` in general like `topicArticles` and `article`

File: components/Cards/index.js [hunk 2]
```diff
@@ -17,3 +17,43 @@
...
+    console.log(error);
+  });
+
+function createArticles(article) {
+  const container = document.createElement("div"),
+    headLine = document.createElement("div"),
+    author = document.createElement("div"),
+    img = document.createElement("div"),
+    imgA = document.createElement("img"),
+    spans = document.createElement("span");
```
Comment (wSedlacek) [inline components/Cards/index.js:39]: Is there any reason you declared them like this with only one `const` rather then a `const` for each one?

File: components/Header/index.js [hunk 3]
```diff
@@ -9,4 +9,25 @@
 //  </div >
 // And add it to the DOM in the .header-container component
 
-function Header() {}
+function header() {
+  let header = document.createElement("div");
+  let date = document.createElement("span");
+  let title = document.createElement("h1");
+  let temp = document.createElement("span");
```
Comment (wSedlacek) [inline components/Header/index.js:16]: Remember to use `const` unless there is a reason you would need to change the variable.

File: components/Header/index.js [hunk 4]
```diff
@@ -9,4 +9,25 @@
 //  </div >
 // And add it to the DOM in the .header-container component
 
-function Header() {}
+function header() {
```
Comment (wSedlacek) [inline components/Header/index.js:12]: Component names are always is pascal case. So this would be `Header()`

File: components/Tabs/index.js [hunk 5]
```diff
@@ -7,3 +7,17 @@
...
+	response.data.topics.forEach((item) => {
+		parent.appendChild(tabComponent(item));
+	});
+});
+
+function tabComponent(data) {
+	let header = document.createElement('div');
+	header.classList.add('tab');
+	header.textContent = data;
+	return header;
```
Comment (wSedlacek) [inline components/Tabs/index.js:22]: Wouldn't a better variable name for this be `tab` rather then `header`?

**Review by wSedlacek — APPROVED** [review]
Overall you have done a good job, there is just some work to be done on code style.

**Pull request merged**